### ISSUE TITLE
bug: Mounting causes app to crash on startup and changes app name to "com.google.android.apps.youtube.app.YouTubeTikTokRoot_Application"

### ISSUE THREAD

**Issue by maybecryptic** [ReVanced/revanced-cli#290]
### Bug description

I have followed all the steps to installed revanced on youtube (current version `18.38.44`) and anytime I try to mount, the command runs successfully, but youtube crashes instantly when trying to launch it on my device, and I noticed the name of the app was changed and it includes "TikTok" in it despite not having the app at all.
![Screenshot_20231012-120305](https://github.com/ReVanced/revanced-cli/assets/6799090/60c48cfd-96ec-4c6c-b68c-baa92b201d8e)

Device; Pixel 6a
OS: Android 12
Rooted with Magisk

Steps to reproduce:
1. Download current version of CLI, patches, integrations, and youtube 18.38.44
2. Install youtube 18.38.44 via adb `adb install youtube_18.38.44.apk`
3. Run patch with mount argument
```
java -jar revanced-cli.jar patch --patch-bundle revanced-patches.jar --out revanced_18.38.44.apk -m integrations.apk --device-serial MYSERIALHERE --mount youtube_18.38.44.apk
```
4. After getting `INFO: Finished installing revanced_18.38.44.apk`, run the app and it will instantly crash. Going into settings > applications will show com.google.android.apps.youtube.app.YouTubeTikTokRoot_Application as well.

### Solution

Patching via the manager works without issue.

### Additional context

I tried removing the microg patch since root doesn't need it, but the result is the same.

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
The app you mounted over has updated and does not match the mounted apk version.

**Comment by maybecryptic** [comment]
> The app you mounted over has updated and does not match the mounted apk version.

How could it update that quickly? I have auto update off.

**Comment by oSumAtrIX** [comment]
I can't say, it's your device; in any case, the installed app and mounted app versions mismatch.

**Comment by maybecryptic** [comment]
So then why does the version match in settings?


https://github.com/ReVanced/revanced-cli/assets/6799090/5dbc512c-14af-402d-85da-c145d75853fd

**Comment by oSumAtrIX** [comment]
It does not match, it displays the mounted apps version.

**Comment by maybecryptic** [comment]
I'm confused - please help me understand. That's the only version of youtube on my device. You're saying I should have two versions - the original and the mounted? I installed the original 18.38.44 on my device and after running the cli patch command, it's replaced with what you see in the screen shot.

![Screenshot_20231012-143350](https://github.com/ReVanced/revanced-cli/assets/6799090/066a0c54-b000-4afc-b249-73cd9abe57c5)

**Comment by maybecryptic** [comment]
Whatever, revanced extended is better updated and does not have this issue.

**Comment by oSumAtrIX** [comment]
Yes it does and the fork has nothing to do with ReVanced Manager.

**Comment by oSumAtrIX** [comment]
Dupe of https://github.com/ReVanced/revanced-manager/issues/237

**Comment by maybecryptic** [comment]
> Yes it does

Sure, whatever you say. I followed the extended docs and it doesn't crash, nor does it have a weird app name.

> the fork has nothing to do with ReVanced Manager

I never said it did

**Comment by oSumAtrIX** [comment]
What you claimed above implies that but is clearly wrong as explained above

**Comment by maybecryptic** [comment]
Let's say I did do something wrong initially - I just went back and tried the same steps with revanced instead of extended and the issue persists. It could be the different version recommendation (revanced is 18.38.44 and extended is 18.36.39), or maybe it has to do with cli itself, or perhaps it's neither. Of course I have no idea what the issue is, and you've been extremely unhelpful getting this troubleshooted. But I just wanted to leave this information in case anyone else runs into this issue.
